demisto_sdk/commands/generate_docs/generate_playbook_doc.py

```diff
@@ -74,7 +74,7 @@ def generate_playbook_doc(input, output: str = None, permissions: str = None, li
             return
 
 
-def get_playbook_dependencies(playbook):
+def get_playbook_dependencies(playbook, input):
     """
     Gets playbook dependencies(integrations, playbooks, scripts and commands) from playbook object.
     :param playbook: the playbook object.
```

Comment:
lets rename `input` to something else, it is a builtin function name and lets add it to the docstring